scylla.go

```diff
@@ -4,17 +4,96 @@ import (
 	"fmt"
 	"math"
 	"strconv"
+	"strings"
```

Comment:
I know these are frustrating but can you put all the formatting in a separate commit?

Reply:
I don't think I follow. What exactly is not right here in terms of formatting?

Reply:
Nothing wrong other than it clutters the patch. I mean where you add a new dependency it's fine but I thought I saw an unrelated formatting. It's sometimes impossible to avoid if upstream also formats strangely. Ignore this if it's too messy or impossible to avoid.

Reply:
But I don't really see any unrelated formatting there, for me it looks aligned just as the rest of the surrounding lines...

Reply:
Maybe you can point me to a place where it's actually wrong?

Reply:
Run fmt on base branch and rebase your work on top.

Reply:
`gofmt` against the base branch shows empty output, so I assume everything was fine before my changes. I'll run another time after I've addressed all the issues and commit the result if there would be anything.